### PR TITLE
Updates to digital ocean machine driver

### PR DESCRIPTION
**Pull request by westlywright**
rancher/rancher#4168

Fix issues with tab selection on load balancer

rancher/rancher#5377

### CODE REVIEW

**Comment by vincent99** [comment]
Requires https://github.com/rancher/cattle/pull/1810

File: app/components/machine/driver-digitalocean/template.hbs [hunk 1]
```diff
@@ -16,79 +14,91 @@
...
+            {{/each}}
+          </select>
+        </div>
+        <div class="col-md-2 form-label">
+          <label class="form-control-static">{{t 'machine.driverDigitalocean.size.label'}}</label>
+        </div>
+        <div class="col-md-4">
+          <select class="form-control" onchange={{action (mut digitaloceanConfig.size) value="target.value"}}>
+            {{#each filteredSizeChoices as |choice|}}
+              <option value={{choice.slug}} selected={{eq digitaloceanConfig.size choice.slug}}>{{choice.slug}}</option>
```
Comment (vincent99) [inline app/components/machine/driver-digitalocean/template.hbs]: Show "# GB RAM, # GB Disk, # vCPU" here since you have the info?

**Comment by vincent99** [comment]
@westlywright Need to catch auth (or other kinds) of error and show them, if you put in a bogus API key it just sits spinning on Loading... forever.

File: app/components/machine/driver-digitalocean/template.hbs [hunk 2]
```diff
@@ -1,7 +1,5 @@
 <section class="horizontal-form">
-  <div class="container-fluid">
-    {{partial "host/add-common"}}
-
+  {{#if step1}}
```
Comment (vincent99) [inline app/components/machine/driver-digitalocean/template.hbs]: Wrap the inside of this `{{#if}}` with a `<form>` tag to appease the 1Password gods.

File: translations/en-us.yaml [hunk 3]
```diff
@@ -2246,6 +2246,7 @@ machine:
 
   driverDigitalocean:
     accountSection: Account Access
+    authAccountButton: Next & Setup Machine
```
Comment (vincent99) [inline translations/en-us.yaml]: Next: Configure Droplet

**Comment by westlywright** [comment]
@vincent99 g2g

**Pull request merged**